### ISSUE TITLE
Poor documentation

### ISSUE THREAD

**Issue by diklou** [filipinascimento/helios-web#22]
The documentation is very incomplete. And there is no type declarations. So it's very hard to simply use helios.

* How do you add labels (tooltips that show up on hover) ?
* and labels that show up when very zoomed in ? (like [this](https://cdn.discordapp.com/attachments/1039793146470596688/1041679726193954906/unknown.png))
* How do you add a different weight to the nodes ?
* How can you customise the color of a node ?
* how can you change the width of an edge ?

**Comment by filipinascimento** [comment]
Thanks for the feedback. The tool is still in early stages of development and indeed the documentation is almost non-existent, making it harder or almost impossible to be used without looking at the source code. We are currently applying for research funds and trying to attract some developers to collaborate in the tool, so that we can move it forward at a faster pace.

I'm still working on the revamped geometry generation system, then the next priority will be documentation, tutorials and more examples.

Regarding your questions:
 - **How do you add labels (tooltips that show up on hover)**
Currently labels are not supported natively. (In the examples, we are adding it on top of the view as a svg layer)

 - **and labels that show up when very zoomed in ?**
Same as above. Real 2D/3D labels in the helios panel still need to be coded

 - **How do you add a different weight to the nodes ?**
Nodes appearance can be customized by changed by using the following functions:

    - `helios.nodeColor(...)`
    - `helios.nodeSize(...)`
    - `helios.nodeOutlineColor(...)`
    - `helios.nodeOutlineWidth(...)`

    The input are like in d3. If you provide a single value, such as an rgb color `helios.nodeColor([1.0,0,0])`, it will set all nodes to red, or in `helios.nodeSize(10.0)`, which sets all nodes' size to 10. If you provide a function instead, you can set the color based on each node information. For instance:

    ```javascript
    helios.nodeColor(node => {
        if(node.category == "dog"){
             return [1,0,0]; // red
        }else{
            return [0,0,1]; // blue
        }
    }).update().render();
    ```

     method `update` updates the geometry of the nodes to account for these changes and `render()` redraws the network with the new attributes.

 - **How can you customise the color of a node ?**
see above

- ** how can you change the width of an edge ? **
This is a feature to be implemented in the next 0.6 update with the revamped geometry system. You will be able to change the width and colors of individual edges.

**Comment by filipinascimento** [comment]
Documentation and support to change edge colors and widths independently from nodes was added in 0.7.
